tests/integration_tests.rs

```diff
@@ -44,20 +46,17 @@ async fn create_offers(
             .await
             .expect("should create offer");
 
-        let messenger_utils = MessengerUtilities::new();
+        // let messenger_utils = MessengerUtilities::new(SEED_SECRET_KEY);
         let blinded_path = offer.paths()[0].clone();
-        let secp_ctx = Secp256k1::new();
-        let reply_path =
-            BlindedPath::new_for_message(reply_path_keys, &messenger_utils, &secp_ctx).unwrap();
-
+        // // // let secp_ctx = Secp256k1::new();
```

Comment:
Did you mean to remove the commented out statements here and elsewhere in the commit?

Reply:
oh I messed up leaving those comments. I'm removing them